src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/Symbols/EENamedTypeSymbol.cs

```diff
@@ -362,9 +362,7 @@ internal override bool IsInterface
         internal override bool IsExtension => false;
         internal override bool IsExplicitExtension => false;
         internal override TypeSymbol GetDeclaredExtensionUnderlyingType() => throw ExceptionUtilities.Unreachable();
-
-        internal override TypeSymbol GetExtendedTypeNoUseSiteDiagnostics(ConsList<TypeSymbol> basesBeingResolved)
-            => throw ExceptionUtilities.Unreachable();
+        internal override TypeSymbol GetExtendedTypeNoUseSiteDiagnostics(ConsList<TypeSymbol> basesBeingResolved) => null;
```

Comment:
>null [](http://example.com/codeflow?start=117&length=4)

Is this always the right value? Even if we are debugging code inside an extension type?